DFC.Composite.Shell/Startup.cs

```diff
@@ -98,7 +98,7 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
                     .CustomSources($"{cdnLocation}/{Constants.NationalCareersToolkit}/images/", "www.google-analytics.com", "*.doubleclick.net"))
                 .ConnectSources(s => s
                     .Self()
-                    .CustomSources($"{Configuration.GetValue<string>(Constants.ApplicationInsightsConnectSources)}", "https://dc.services.visualstudio.com/")));
+                    .CustomSources($"{Configuration.GetValue<string>(Constants.ApplicationInsightsConnectSources)}", "https://dc.services.visualstudio.com/", "https://*.api.nationalcareersservice.org.uk/")));
```

Comment:
This will have to come from appSettings, this value will be different for each environment.